File: ApiSuvesaPos/Datos/Class/CajasCantidad.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Datos.Models;
using Microsoft.EntityFrameworkCore;

namespace Datos.Class
{
    class CajasCantidad
    {
        SeePOSContext entities;

        public CajasCantidad()
        {
            entities = new SeePOSContext();
        }


        public List<Models.CajasCantidad> ObtenerCajasCantidad()
        {
            try
            {
                List<Models.CajasCantidad> result;


                    var temp = from c in entities.CajasCantidads
                               where c.NumApertura == 0
                               select c;

                    result = temp.ToList<Models.CajasCantidad>();
              

                if (result.Count > 0)
                {
                    return result;
                }
                else
                {
                    return result = null;
                }

            }
            catch (Exception ex)
            {
                throw ex;
            }
        }

    }
}

File: ApiSuvesaPos/Negocio/Logica/Cajas_Cantidad.vb
Namespace Logica
    Public Class Cajas_Cantidad

        Private db As Datos.Class.CajasCantidad

        Sub New()
            Me.db = New Datos.Class.CajasCantidad
        End Sub

        Public Function Buscar() As List(Of Datos.Models.CajasCantidad)
            Return Me.db.ObtenerCajasCantidad
            'Me.AdapterNumCaja.Fill(Me.DataSetAperturaCaja1.Cajas_Cantidad)

        End Function

    End Class
End Namespace